### PR TITLE
Feat(client/angular-marketplace): Initialization of the project with readme.

### PR DESCRIPTION
**Pull request by jeannsensei**
I finished the readme and creation of the client side project via Angular CLI. 
The next thing to discuss is:  Which libraries are we going to use? (in the DAT project if I'm not mistaken, they use Angular Material).

### CODE REVIEW

**Comment by jeannsensei** [comment]
This PR is related to: #1

**Review by davidtorroija — APPROVED** [review]
Ok this looks good!

**Comment by davidtorroija** [comment]
Ok the PR looks good testing testing

File: client/angular-marketplace/.gitignore [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+/connect.lock
+/coverage
+/libpeerconnection.log
+npm-debug.log
+yarn-error.log
+testem.log
+/typings
+
+# System Files
+.DS_Store
```
Comment (davidtorroija) [inline client/angular-marketplace/.gitignore:44]: Testing comment

**Pull request merged**